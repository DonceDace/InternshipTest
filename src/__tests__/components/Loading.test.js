import Loading from '@/components/loading';
import { render } from '@testing-library/react';

describe('Component: Loading', function () {
  it('match snapshot', async function () {
    const result = render(<Loading />).container.outerHTML;
    expect(result).toMatchSnapshot();
  });
});
