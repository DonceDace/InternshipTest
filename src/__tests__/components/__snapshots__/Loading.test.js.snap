// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Loading match snapshot 1`] = `"<div><div style=\\"text-align: center;\\"><img style=\\"width: 2rem; height: 2rem;\\" src=\\"https://file-storage.distinctclinic.com/distinct-wxmp/static/img/loading2.gif\\" alt=\\"loading\\"></div></div>"`;
